Examples/Sample_xServiceSet.ps1

```diff
@@ -1,20 +1,27 @@
-﻿# Starts a set of existing stopped services
+﻿<#
+    .SYNOPSIS
+        Sets the services with the given names to start under the built-in account LocalService.
 
-Configuration xServiceSetExample
+    .PARAMETER ServiceNames
+        The names of the services to set.
+#>
+Configuration xServiceSetBuiltInAccountExample
 {
-    param (
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
         [String[]]
-        $StoppedServiceNames
+        $ServiceNames
```

Comment:
This and the other examples are a bit theoretical as there is no actual value for services.  Take this example, the whole point to to say that you can pass multiple values to ServicesNames.  Unless you happen to notice that the parameter is an array, it does not seem to be the focus of the example.  It seems to focus on setting a service to a built-in account.